plugins/win-mf/mf-h264.cpp

```diff
@@ -250,20 +250,20 @@ static void MFH264_GetDefaults(obs_data_t *settings)
 #define PROP_DEF(x, y, z) obs_data_set_default_ ## x(settings, y, z)
 	PROP_DEF(int,    MFP_BITRATE,         2500);
 	PROP_DEF(bool,   MFP_USE_LOWLAT,      true);
-	PROP_DEF(int,    MFP_B_FRAMES,        2);
+	PROP_DEF(int,    MFP_B_FRAMES,        0);
 	PROP_DEF(bool,   MFP_USE_BUF_SIZE,    false);
 	PROP_DEF(int,    MFP_BUF_SIZE,        2500);
 	PROP_DEF(bool,   MFP_USE_MAX_BITRATE, false);
 	PROP_DEF(int,    MFP_MAX_BITRATE,     2500);
 	PROP_DEF(int,    MFP_KEY_INT,         2);
-	PROP_DEF(int,    MFP_RATE_CONTROL,    H264RateControlCBR);
+	PROP_DEF(int,    MFP_RATE_CONTROL,    H264RateControlVBR);
```

Comment:
Question, why are we changing the defaults here vs passing in different options from our side?

Reply:
Passing in options from our side doesn't actually update the media foundation encoder. It seems like they get set during initialization and then the update method doesn't update it appropriately. I am actually not sure if the current encoder updates work either. This basically was just a faster solution.